frontend/src/lib/OutputArtifactLoader.ts

```diff
@@ -472,10 +472,11 @@ async function getExecutionInContextWithPodName(
     return undefined; // Not found, this is expected to happen normally when there's no mlmd data.
   }
   const state = foundExecution.getPropertiesMap().get('state');
-  if (!state || state.getStringValue() !== 'complete') {
-    return undefined; // Execution doesn't have a valid state, or it has not finished.
+  if (state && (state.getStringValue() === 'complete' || state.getStringValue() === 'cached')) {
```

Comment:
```suggestion
  if (['complete', 'cached'].includes(state?.getStringValue()) {
```
nit: can be simplified a little with some new syntax -- [optional chaining](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Optional_chaining)

Reply:
Done. Thx